src/index.js

```diff
@@ -34,6 +34,24 @@ function parsePart(part) {
   });
 }
 
+function validatePart(timeElementValues, validator){
+  let elements = [], isValid = true;
+
+  if(timeElementValues != null) {
+    timeElementValues.forEach((value)=> {
+      elements = elements.concat(Object.values(value))
+    });
+
+    isValid = elements.every((element)=> element >= validator.min && element <= validator.max );
+
+    if(!isValid){
+      throw new Error(
+        `Invalid time element. Range should be within [${validator.min} - ${validator.max}].`
```

Comment:
I think "time range" is more accurate than "time element". What do you think?

Reply:
Sure :)